health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/RenderSelectionField.js

```
@@ -4,7 +4,7 @@ import { Loader, SelectionTag } from "@egovernments/digit-ui-components";
 const RenderSelectionField = ({ field, t }) => {
   const isMdmsEnabled = field?.isMdms && !!field?.schemaCode;
 
-  const { isLoading, data } = Digit?.Hooks.useCustomMDMSuseCustomMDMS(
+  const { isLoading, data } = Digit?.Hooks.useCustomMDMS(
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)